RELEASES.md

```diff
@@ -6,21 +6,31 @@ Version 1.82.0 (2024-10-17)
 Language
 --------
 - [Don't make statement nonterminals match pattern nonterminals](https://github.com/rust-lang/rust/pull/120221/)
-- [Implement lint against ambiguous negative literals](https://github.com/rust-lang/rust/pull/121364/)
 - [Patterns matching empty types can now be omitted in common cases](https://github.com/rust-lang/rust/pull/122792)
+- [`addr_of(_mut)!` macros and the newly stabilized `&raw (const|mut)` are now safe to use with all static items](https://github.com/rust-lang/rust/pull/125834)
 - [size_of_val_raw: for length 0 this is safe to call](https://github.com/rust-lang/rust/pull/126152/)
 - [Reorder trait bound modifiers *after* `for<...>` binder in trait bounds](https://github.com/rust-lang/rust/pull/127054/)
```

Comment:
We still need to update the reference grammar here: https://doc.rust-lang.org/nightly/reference/trait-bounds.html#trait-and-lifetime-bounds

Reply:
Can you file a separate issue or otherwise track that? This doesn't seem like something relevant to the release notes?

Reply:
The reference is updated now: https://github.com/rust-lang/reference/pull/1650.